<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>583</width>
    <height>393</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,1">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Your message:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="SendToAllMessageButton">
        <property name="text">
         <string>SendToAll</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="SendPrivateMessageButton">
        <property name="text">
         <string>Send Private</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Getuserslist</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>                                   Chat                                                                       Private Chat   </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTextBrowser" name="commonChatBrowser"/>
      <widget class="QTextBrowser" name="privateChatBrowser"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>583</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMain_menu">
    <property name="title">
     <string>Main menu</string>
    </property>
    <addaction name="actionOpen_enother_client"/>
    <addaction name="actionClose_this_client"/>
   </widget>
   <addaction name="menuMain_menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen_enother_client">
   <property name="text">
    <string>Open enother client</string>
   </property>
  </action>
  <action name="actionClose_this_client">
   <property name="text">
    <string>Close this client</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
